### ISSUE TITLE
Python 3.10 / Janus 0.6.2

### ISSUE THREAD

**Issue by holgerpieta** [Azure/azure-iot-sdk-python#891]
# Context

- **OS and version used:** Windows 10
- **Python version:** 3.10
- **pip version:** 21.3.1
- **list of installed packages:**
aiohttp             3.7.4
async-timeout       3.0.1
attrs               21.2.0
azure-iot-device    2.8.0
certifi             2021.10.8
chardet             3.0.4
charset-normalizer  2.0.7
deprecation         2.1.0
idna                3.3
janus               0.4.0
multidict           5.2.0
packaging           21.2
paho-mqtt           1.6.1
pip                 21.3.1
pyparsing           2.4.7
PySocks             1.7.1
requests            2.26.0
requests-unixsocket 0.2.0
setuptools          57.4.0
six                 1.16.0
toshiba-ac          0.1.12
typing-extensions   3.10.0.2
urllib3             1.26.7
yarl                1.7.0
- **cloned repo:** N/A

## Description of the issue

Janus < 0.6.2 does not work with Python 3.10 due to breaking changes in asyncio: https://docs.python.org/3/whatsnew/3.10.html#changes-in-the-python-api
This is fixed in 0.6.2.
azure-iot requires Janus 0.4.0, so it does not work.

## Code sample exhibiting the issue

Run the sample GUI in https://github.com/KaSroka/Toshiba-AC-control

## Console log of the issue

The GUI above is a tkinter program, so console logs are a bit of a problem, but I managed to trace it back to:
```
azure\iot\device\iothub\aio\async_inbox.py", line 35, in make_queue
    return janus.Queue()
janus\__init__.py", line 39, in __init__
    self._async_mutex = asyncio.Lock(loop=loop)
```

**Comment by cartertinney** [comment]
@holgerpieta

Thanks for pointing this out - we haven't tried this out in Python 3.10 yet, so we'll look into getting this addressed as soon as possible

**Comment by cartertinney** [comment]
@holgerpieta This has been addressed in #892 and will be included as part of the next release

**Comment by cartertinney** [comment]
@holgerpieta This has been included as part of azure-iot-device 2.9.0, now available via pip
